app/retail/templates/jtbd/earn.html

```diff
@@ -1,9 +1,312 @@
+            </div>
+  
+            <h5 class="accordion collapsed" id="headingTwo" data-parent="#accordion-group" data-toggle="collapse" data-target=".collapseTwo">
+              <span class="accordion-title">Earn through sponsored hackathons</span>
+            </h5>
+            <div class="accordion-panel collapse font-subheader text-grey-500 collapseTwo" aria-labelledby="headingTwo" data-parent="#accordion-group">
+              <p class="font-header font-bigger-1">Collect rewards and tokens as you hack your way through creative solutions on a variety of protocols. Team up with other Web3 builders to advance the technology that will shape our future.</p>
+            </div>
+  
+            <h5 class="accordion collapsed" id="headingThree" data-parent="#accordion-group" data-toggle="collapse" data-target=".collapseThree">
+              <span class="accordion-title">Earn through grant funding</span>
+            </h5>
+            <div class="accordion-panel collapse font-subheader text-grey-500 collapseThree" aria-labelledby="headingThree" data-parent="#accordion-group">
+              <p class="font-header font-bigger-1">Start an open source, public goods project and begin crowdsourcing funds through Gitcoin Grants. Participate in our quarterly Grants Rounds for the opportunity to earn a portion of the generous matching pool that has helped launch thousands of projects.</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section id="top-earners">
+      <h2 class="text-grey-500 mb-5">See top earners on Gitcoin</h2>
+      <div class="row pt-3 mb-0">
+        {% for earner in top_earners %}
+        <div class="col-12 col-md-6 col-lg-3 mb-5">
+          <div class="row">
+            <div class="col-5 col-md-12">
+              <img class="rounded-circle w-100" style="max-width: 10rem;" src="{% url 'org_avatar' earner.github_username %}" />
+            </div>
+            <div class="col-7 col-md-12 pt-2 m-auto">
+              <div class="row pl-md-4 mt-3">
+                <div class="col-12 col-md-2 p-0">
+                  <h2 class="d-inline w-100 gc-font-base text-grey-500 p-0">#{{ earner.rank }}</h2>
+                </div>
+                <div class="col-12 col-md-8 pt-2 pl-md-2 pl-0" style="line-height: 10px;">
+                  <p class="w-100 text-grey-400 font-subheader">@{{ earner.github_username }}</p>
+                  <p class="w-100 text-grey-400 font-subheader">{{ earner.amount|floatformat:"0" }} DAI</p>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        {% endfor %}
+      </div>
+      <a class="text-primary font-bigger-1" href="{{ BASE_URL }}/leaderboard/earners?cadence=monthly&keyword=all&product=bounties" target="_blank">View Leaderboard</a>
+    </section>
+
+    <section id="hackathons">
```

Comment:
This shows active hackathons right ?  would it make sense to add a v-if to this ?

Reply:
oh yes... nice catch!